src/xdocs/config_misc.xml

```diff
@@ -1320,6 +1320,111 @@ void foo(String aFooString,
       </subsection>
     </section>
 
+    <section name="OrderedProperties">
+      <subsection name="Description" id="OrderedProperties_Description">
+          <p>Since Checkstyle 5.8</p>
```

Comment:
We are working towards 8.20, you should probably put 8.21 as I don't think this will done before the release next Saturday.

Reply:
Done.